queue_services/entity-filer/src/entity_filer/filing_processors/change_of_registration.py

```diff
@@ -86,121 +89,121 @@ def post_process(business: LegalEntity, filing: Filing):
-    ) and (naics_code := naics.get("naicsCode")):
-        if legal_entity.naics_code != naics_code:
-            filing_meta.change_of_registration = {
-                **filing_meta.change_of_registration,
-                **{
-                    "fromNaicsCode": legal_entity.naics_code,
-                    "toNaicsCode": naics_code,
-                    "naicsDescription": naics.get("naicsDescription"),
-                },
-            }
-            legal_entity_info.update_naics_info(legal_entity, naics)
-
-
-def _update_sp_change(
-        legal_entity: LegalEntity,
-        change_filing_rec: Filing,
-        change_filing: Dict,
-        filing_meta: FilingMeta,
-):
-    name_request = dpath.util.get(change_filing, "/changeOfRegistration/nameRequest", default=None)
-    identifier = dpath.util.get(change_filing_rec.filing_json, "filing/business/identifier")
-    if name_request and (to_legal_name := name_request.get("legalName")):
-        alternate_name = AlternateName.find_by_identifier(identifier)
-        parties_dict = dpath.util.get(change_filing, "/changeOfRegistration/parties")
-
-        # Find the Proprietor
-        proprietor = None
-        for party in parties_dict:
-            for role in party.get("roles"):
-                if role.get("roleType") == "Proprietor":
-                    proprietor_dict = party
-                    break
-            if proprietor_dict:
-                break
-
-        if not proprietor_dict:
-            raise DefaultException(
-                f"No Proprietor in the SP registration for filing:{change_filing_rec.id}"
-            )
-
-        proprietor, delivery_address, mailing_address = get_or_create_party(
-            proprietor_dict, change_filing_rec
-        )
-        if not proprietor:
-            raise DefaultException(
-                f"No Proprietor in the SP registration for filing:{change_filing_rec.id}"
-            )
-
-        alternate_name.end_date = change_filing_rec.effective_date
-        alternate_name.change_filing_id = change_filing_rec.id
-        # alternate_name.delete()
-        db.session.add(alternate_name)
-        db.session.commit()
-        db.session.delete(alternate_name)
-        db.session.commit()
-
-        new_alternate_name = AlternateName(
-            identifier=identifier,
-            name_type=AlternateName.NameType.OPERATING,
-            change_filing_id=change_filing_rec.id,
-            end_date=None,
-            name=to_legal_name,
-            start_date=alternate_name.start_date,
-            registration_date=change_filing_rec.effective_date,
-        )
-        proprietor.alternate_names.append(new_alternate_name)
-
-        filing_meta.change_of_registration = {
-            **filing_meta.change_of_registration,
-            "fromLegalName": alternate_name.name,
-            "toLegalName": to_legal_name,
-        }
+# def _update_partner_change(
+#         legal_entity: LegalEntity,
+#         change_filing_rec: Filing,
```

Comment:
can all of the commented out code be removed?

Reply:
Yes, I will do.
But, I just wanted you to confirm my changes before I remove the commented-out code.